platform-ajax/src/main/typescript/com/softicar/platform/ajax/engine/request/AjaxRequestMessage.ts

```diff
@@ -90,7 +90,55 @@ class AjaxRequestMessage {
 		return new AjaxRequestMessageEncoder(this.data).encodeToHex();
 	}
 
-	// ------------------------------ private ------------------------------ //
+	// ------------------------------ redundancy ------------------------------ //
+
+	public isRedundantTo(other: AjaxRequestMessage) {
```

Comment:
this method defines which requests will be dropped if other requests are queued or sent